pages/apim/3.x/installation-guide/docker/installation-guide-docker-images.adoc

```diff
@@ -1,28 +1,24 @@
-= Docker Image Installation
+= Docker Images Install
 :page-sidebar: apim_3_x_sidebar
 :page-permalink: apim/3.x/apim_installation_guide_docker_images.html
 :page-folder: apim/installation-guide/docker
 :page-layout: apim3x
 :page-description: Gravitee.io API Management - Installation Guide - Docker - Images
 :page-keywords: Gravitee.io, API Management, apim, guide, manual, docker, images, linux
-:page-toc: false
 :page-liquid:
 :table-caption!:
 
-// author: Tom Geudens
+This page describes how to install and run APIM in Docker containers on `localhost` using the `docker` command and a specified filesystem for persistence and plugins. 

```

Comment:
Consider providing context as to why a user might want to pull and run the images separately as opposed to just using docker-compose.

Reply:
I think the only reason is personal preference, and I don't think it is necessary to make that explicit.